.snyk

```diff
@@ -0,0 +1,6 @@
+ignore:
+  MPL-2.0
+    - reason: 'MPL-2.0 is compatible with Rook licenses'
```

Comment:
For clarity when communicating behavior, avoiding acronyms is best so that others (or our future selves) don't need to google search to reference simple things.
```suggestion
    - reason: 'Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license'
```